packages/experimental/Expander/windows/ReactNativeExpander/ExpanderView.cpp

```diff
@@ -0,0 +1,255 @@
+#include "pch.h"
+#include "JSValueXaml.h"
+#include "ExpanderView.h"
+#include "ExpanderView.g.cpp"
+
+#include <limits>
+#include <stdexcept>
+
+namespace winrt {
+    using namespace Microsoft::ReactNative;
+    using namespace Windows::Foundation;
+    using namespace Windows::UI::Xaml::Media;
+}
+
+namespace winrt::ReactNativeExpander::implementation {
+
+    ExpanderView::ExpanderView(winrt::IReactContext const& reactContext) : m_reactContext(reactContext) {
+        auto expander = Microsoft::UI::Xaml::Controls::Expander();
+        this->VerticalContentAlignment(Windows::UI::Xaml::VerticalAlignment::Top);
+        this->Content(expander);
+
+        RegisterEvents();
+    }
+
+    void ExpanderView::RegisterEvents() {
+        auto expander = (this->Content()).try_as<Microsoft::UI::Xaml::Controls::Expander>();
```

Comment:
`->` and `.` operators have the same precedence and are evaluated LTR in C++, so you can remove the parentheses around Content